modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -516,6 +594,35 @@ cv::GArg IECallContext::packArg(const cv::GArg &arg) {
     }
 }
 
+cv::MediaFrame* IECallContext::prepareKeepAliveFrameSlot(req_key_t key) {
+    std::lock_guard<std::mutex> lock(keep_alive_frames_mutex);
+    return &keep_alive_pp_frames[key];
+}
```

Comment:
`*` is way to C-ish, Y it can't be `&`? I don't see you returning `nullptr` here so not clear what is the case for `*`.